yorkie/packs/packs.go

```diff
@@ -329,14 +331,21 @@ func pullSnapshot(
 		return nil, nil, err
 	}
 
-	if err := doc.ApplyChangePack(change.NewPack(
-		docKey,
-		checkpoint.Initial.NextServerSeq(docInfo.ServerSeq),
-		changes,
-		nil,
-	)); err != nil {
-		return nil, nil, err
-	}
+	var startIdx, endIdx = 0, CHANGE_SEGMENT_SIZE;
```

Comment:
I haven't verified the logic of this, just wanted to make sure that this is indeed the right direction, before further implementation and testing